tests/test_hf_api.py

```diff
@@ -434,6 +435,28 @@ def test_dataset_info(self):
         self.assertEqual(dataset.sha, DUMMY_DATASET_ID_REVISION_ONE_SPECIFIC_COMMIT)
 
 
+class HfApiPrivateTest(HfApiCommonTestWithLogin):
+    def setUp(self) -> None:
+        super().setUp()
+        self._api.create_repo(token=self._token, name=REPO_NAME)
```

Comment:
nit: alternatively you could use `create_repo` with `private=True` since the get go.

Reply:
Thanks for the tip - done!